doc/classes/String.xml

```diff
@@ -328,7 +336,14 @@
 		<method name="is_valid_float" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if this string contains a valid float.
+				Returns [code]true[/code] if this string contains a valid float. This is inclusive of integers, and also supports exponents:
+				[codeblock]
+				print("1.7".is_valid_float()) # Prints "true"
+				print("24".is_valid_float()) # Prints "true"
+				print("7e3".is_valid_float()) # Prints "true"
+				print("24".is_valid_float()) # Prints "true"
+				print("B".is_valid_float()) # Prints "false"
```

Comment:
For the sake of readability, let's choose text that doesn't look like the number 8.

```suggestion
				print("Hello".is_valid_float()) # Prints "false"
```

Reply:
Good catch, I'll get to it shortly.

Reply:
Ok, done.